edge-functions/crypto/next-env.d.ts

```diff
@@ -1,5 +1,5 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
+/// <reference types="next/dist/styled-jsx-types/global" />
```

Comment:
Was this a manual change? Next.js should be removing this line entirely 🤔

Reply:
See https://github.com/vercel/next.js/pull/37902

Reply:
After https://github.com/vercel/next.js/pull/37964 it can be removed now.